lock/src/main/java/com/auth0/lock/Lock.java

```diff
@@ -362,6 +367,7 @@ public static Lock getLock(Activity activity) {
         private boolean useEmail;
         private String defaultDBConnectionName;
         private List<String> connections;
+        private List<String> enterpriseConnectionsUsingWebForm;
```

Comment:
So we can avoid https://github.com/auth0/Lock.Android/pull/159/files#diff-035023a6fd0804236699900c41690be7R258